cli/dcoscli/data/help/experimental.txt

```diff
@@ -0,0 +1,51 @@
+Description:
+    Experimental commands. These commands are under development and are subject to change.
```

Comment:
Is there a JIRA ticket to track and read about the motivation.
* Experimentation via beta-build is a common/ well-known approach in projects that are open source.
* Adding a sub-command to public releases will make people rely on it and will be hard to go back on experiments. ( I hope that was vetted out).